ai-platform-unified/notebooks/unofficial/sdk/metric-parameter-tracking-for-locally-trained-models.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>Please add cleanup code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>Actually there is no other clean up code here other than delete the project. For preview, we actually do not allow user to delete metadataStores and resources directly(this is partly because metadata service is free during preview launch), so other than delete the project, we would not be able to fully delete the metadata. </p>